client/stylesheets/shared/_embed.scss

```diff
@@ -125,3 +125,17 @@
 		}
 	}
 }
+
+// for the embed page with the new navigation enabled
+.woocommerce-feature-enabled-navigation.woocommerce-embed-page {
```

Comment:
I think `has-woocommerce-navigation` is a better class to use since it is more explicit that the nav is on the existing page and not just that the feature is enabeld.